### ISSUE TITLE
Entrance/exit from collection not supported by ScrollView

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30967]
## Description
Entrance/exit from collection not supported by ScrollView. No components can be set as any collection role. With support for "List" or "Grid" roles would these would likely work as expected.

This issue is covered in-depth by parent issue #30861, please check there for more details.

## React Native version:
0.63

**Comment by mdaj06** [comment]
@amarlette can i have a look at this?

**Comment by joachimwedin** [comment]
Hi @amarlette, I would like to contribute to this issue, could you please assign it to me?

**Comment by pablolucas890** [comment]
Hello @amarlette , I see #30861 is closed, isn't better closed this issue too?

**Comment by XantreGodlike** [comment]
I think this issue should be closed. Can confuse newbies, like me

**Comment by fabriziobertoglio1987** [comment]
video test case included here https://github.com/facebook/react-native/pull/34267
